Checklist Overhaul

MeemawHustlin opened the pull request:
Mainly added icons to the checklist, but also updated the overall organization of the checklist in various ways.

MeemawHustlin committed:
Added checklist icons

Adding icons seems to have resulted in a few issues I'll be fixing soon:

- Checklist elements are not highlighting green correctly.
- Certain columns such as closing random oblivion gates are now stuck forced into a single column for some reason.

MeemawHustlin committed:
Fixed checklist issue

It turns out specifically renaming the "40 Random Gates" subsection in misc.json will break the entire checklist functionality, so I just had to switch the name of this one back to get it working again. This behavior may be worth fixing later but I just used a note to explain the random gates instead.

MeemawHustlin committed:
Added overview checklist icons to table

MeemawHustlin committed:
Added missing icon files

MeemawHustlin committed:
Added css for overview icons

MeemawHustlin committed:
Fixed progress tracking incorrectly

parseInt was changed to parseFloat to allow for objective weights with decimal values.

MeemawHustlin committed:
Added fame icon, removed Namira from Fame

Namira gave 0 fame, so her checklist item has been removed to avoid confusion.

MeemawHustlin committed:
Updated checklist organization

- Renamed sections to match in-game objective names.
- Reordered the overview at the top of the checklist to be the order the objectives appear in-game.
- Reordered the order of the objectives on the checklist to be closer to in-game order without sacrificing navigation flow.
- Added more sub-sections to oblivion gate places, and nirnroot.
- Named all the indoor/city nirnroot to "Nirnroot" to clean up messy looking formIDs.

MichaelEbert commented on main.css:
```diff
@@ -46,6 +46,12 @@ td{
 	padding:0.2em;
 }
 
+.checklist-icon{
```
why is this 18 px, but in the miscChecklistStuff code, it's 16 px? also, we should use em instead of px, so it works with different text sizes. (1 em = width of the "m" character)

MeemawHustlin committed:
Made requested changes

Removed an extra </span>, changed padding/width values to be em instead of px, and made icons on overview the same size as the ones in the actual checklist.
